src/main/java/jenkins/advancedqueue/JobGroup.java

```diff
@@ -100,9 +123,40 @@ public void setPriorityStrategy(PriorityStrategy priorityStrategy) {
 	@Deprecated
 	private String jobPattern = ".*";
 	private boolean usePriorityStrategies;
-	private List<JobGroup.PriorityStrategyHolder> priorityStrategies = new ArrayList<JobGroup.PriorityStrategyHolder>();
+	// private List<JobGroup.PriorityStrategyHolder> priorityStrategies = new ArrayList<JobGroup.PriorityStrategyHolder>();
+	private List<PriorityStrategy> priorityStrategies = new ArrayList<PriorityStrategy>();
 
-	private JobGroup() {
+	@DataBoundConstructor
+	public JobGroup(int id, String description, int priority, JobInclusionStrategy jobGroupStrategy, boolean runExclusive, boolean usePriorityStrategies, List<? extends PriorityStrategy> priorityStrategies) {
```

Comment:
It does not look like you use this constructor anywhere except data binding. For data binding there is no need to repeat fields in the constrictor if you have `@DataBoundSetter`s defined